docs/topics/technology-preview-feature.adoc

```diff
@@ -1,5 +1,6 @@
 // When including this file, ensure that {FeatureName} is set immediately before the include. Otherwise it will result in an incorrect replacement.
 
+:_content-type: SNIPPET
```

Comment:
@stoobie @sbeskin-redhat  As it stands, this is a module, since it has an ID

Reply:
@RichardHoch @sbeskin-redhat 
If it's include within multiple modules to avoid writing the same text repeatedly, then it's a snippet. If in that case, yes, you can remove the ID.
I sometimes wondered myself about whether to include an ID in a snippet, but there's really no use for it I can think of.